### DISCUSSION TITLE
How to solve noice/lsp/signature.lua:47: attempt to call field 'get_clients' (a nil value)?

### DISCUSSION THREAD

**Discussion by nurlan-aliyev** [LazyVim/LazyVim#3182] [Q&A]
Just updated to LazyVim 11.0.1 and whenever I open it I receive this error popup. 
I checked for updates, cleaned noice.nvim and reinstalled, but still a problem.

Help much appreciated,
Thanks!
![image](https://github.com/LazyVim/LazyVim/assets/19326995/dcc01ab0-89eb-487e-93b1-11040e3ddafa)

**Comment by dpetka2001** [comment]
What is your Neovim version?

**Reply by nurlan-aliyev** [reply]
v10

**Comment by folke** [comment]
Just fixed it. My bad

**Reply by nurlan-aliyev** [reply]
Thanks! All fine now
